submitter/base.py

```diff
@@ -56,6 +59,14 @@ def send_entity(self, entity: Entity, entity_type: str, error_key: str,
             entity.add_error(error_key, error_msg)
             return ERRORED_ENTITY, accession
 
+    def __get_accession(self, entity, entity_type):
+        accession = entity.get_accession(entity_type)
+        # TODO We have to do it because of our inconsistant schema.
+        # This should be moved to submission_broker when we set the accession
+        if isinstance(accession, list):
+            accession = accession[0]
+        return accession
+
```

Comment:
why not use the first_element_or_self function here?

Reply:
It is funny. I just did that in my new PR in a method above, but you are right. It should be here, so I already committed that fix. Thanks